server/test/suite/config.test.mjs

```diff
@@ -24,22 +24,23 @@ import { config } from "../../src/config.mjs";
 
 test(`config works properly`, () => {
   const expectedResult = {
+    validate: ["method", "params", "response", "events"],
```

Comment:
It looks like there are a large number of unnecessary changes in this file.

Reply:
Hey Kurt, the only actual change in this file is line no: 28 and line no:43, wherein the test case was updated for including our new configuration - multiUserConnections: "warn"
The rest of the changes are just re-arranging the existing kv pairs a bit (no existing value has been updated/deleted), if you take a closer look
The reason is when I run this test locally, the test passes with just the changes in 28 and 43. But when the same test was run from concourse as part of PR, the response of 'config' had the fields differing in order, hence changed the order in the test

Reply:
No worries. I knew it was mostly formatting but I didn't know there was a reason behind some of it.